Remotion/Core/Reflection.CodeGeneration.UnitTests/ClassEmitterTest.cs

```diff
@@ -20,6 +20,7 @@
 using Castle.DynamicProxy.Generators.Emitters;
 using Castle.DynamicProxy.Generators.Emitters.SimpleAST;
 using NUnit.Framework;
+using Remotion.Development.UnitTesting.NUnit;
```

Comment:
There are a lot of leftover changes due to the revert where this using import is the only change in the file.